src/oops/SOLID/singleResponsibilityPrinciple/before/model/Employee.java

```diff
@@ -63,31 +63,11 @@ public String getFullName(){
         return this.firstName + " " + this.lastName;
     }
 
-    public static void save(Employee employee){
-        try {
-            StringBuilder sb = new StringBuilder();
-            sb.append("### EMPLOYEE RECORD ####");
-            sb.append(System.lineSeparator());
-            sb.append("NAME: ");
-            sb.append(employee.firstName + " " + employee.lastName);
-            sb.append(System.lineSeparator());
-            sb.append("POSITION: ");
-            sb.append(employee.getClass().getTypeName());
-            sb.append(System.lineSeparator());
-            sb.append("EMAIL: ");
-            sb.append(employee.getEmail());
-            sb.append(System.lineSeparator());
-            sb.append("MONTHLY WAGE: ");
-            sb.append(employee.monthlyIncome);
-            sb.append(System.lineSeparator());
+	public String getFirstName() {
+		return firstName;
+	}
 
-            Path path = Paths.get(employee.getFullName()
-                    .replace(" ","_") + ".rec");
-            Files.write(path, sb.toString().getBytes());
-
-            System.out.println("Saved employee " + employee.toString());
-        } catch (IOException e){
-            System.out.println("ERROR: Could not save employee. " + e);
-        }
-    }
+	public String getLastName() {
+		return lastName;
+	}
```

Comment:
Instead of adding this, you can use getFullName() in the caller, it will return firstName+" "+lastName

Reply:
I realized this when I was reviewing your PR but didn't want to change it before you reviewed mine :)